### PR TITLE
[OYPD-301] Upgrade path for alerts.

### PR DESCRIPTION
**Pull request by Kerby70**
Steps for review
- [x] Install profile prior to 8.x-0.2
- [x] Run update process
- [x] See Alert CT on /node/add
- [x] See Alerts Blocks placed in header & pre footer on /admin/structure/block

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
<strong>2</strong>  errors for CodeSniffer: Drupal standard file http://openy-dev.ffwua.com/build579/build_reports/Drupalsniff.txt
Build site installed at http://openy-dev.ffwua.com/build579

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/579/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build579/profiles/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build580

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/580/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build580/profiles/openy/build/reports/behat

File: openy.install [hunk 1]
```diff
@@ -40,3 +40,15 @@ function openy_update_8003() {
     $importer->import('openy_demo_node_landing');
   }
 }
+
+/**
+ * Enable Alerts.
+ */
+function openy_update_8004() {
```
Comment (alexschedrov) [inline openy.install]: Please merge with master and set 8002. @hamrant has been fixed number for his hook_update_N https://github.com/ymcatwincities/openy/pull/181#pullrequestreview-23729058.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

File: openy.install (hunk 1 above)
Comment (Kerby70) [reply openy.install]: @Sanchiz Change committed.

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
Build site installed at http://openy-dev.ffwua.com/build582

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/582/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build582/profiles/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**